apps/impala/src/impala/server.py

```diff
@@ -28,7 +28,7 @@
 
 from ImpalaService import ImpalaHiveServer2Service
 from impala.impala_flags import get_webserver_certificate_file, is_webserver_spnego_enabled, is_kerberos_enabled
-from impala.conf import DAEMON_API_USERNAME, DAEMON_API_PASSWORD, COORDINATOR_URL
+from impala.conf import AUTH_USERNAME, AUTH_PASSWORD, DAEMON_API_USERNAME, DAEMON_API_PASSWORD, COORDINATOR_URL, USE_HTTP_BASIC_AUTH
```

Comment:
Note that AUTH_USERNAME, AUTH_PASSWORD when specified and USE_SASL=false (default IIRC) will be used by the Hue server to authenticated with the Impalad to submit queries via Thrift.

Is it safe that we should use the same credentials for communicating with the WebUi and the Thrift server?